src/schemathesis/schemas.py

```diff
@@ -263,6 +263,22 @@ def spec_version(self) -> str:
     def verbose_name(self) -> str:
         return f"Open API {self.spec_version}"
 
+    @property
+    def base_path(self) -> str:
+        """Base path for the schema."""
+        servers = self.raw_schema.get("servers", [])
+        if servers:
+            # assume we're the first server in list
+            server = deepcopy(servers[0])
+            server_vars = server.pop("variables", {})
+            server["url"] = server["url"].format(**{k: v["default"] for k, v in server_vars.items()})
```

Comment:
do we need this assignment in the dict? it seems like `deepcopy` is not needed as well